core/src/test/resources/log4j.xml

```diff
@@ -101,14 +117,31 @@
     <appender-ref ref="PENTAHOFILE"/>
   </category>
 
+  <!-- ======================= -->
+  <!-- Setup the category for test goal -->
+  <!-- ======================= --> 
+  <category name="org.pentaho.platform.engine.security.event">
+    <priority value="DEBUG"/>
+    <appender-ref ref="PENTAHOCONSOLEDEBUG"/>
+  </category>
+  
+  <category name="org.pentaho.platform.util.logging.Logger">
+    <priority value="DEBUG"/>
+    <appender-ref ref="PENTAHOCONSOLEDEBUG"/>
+  </category>
+  
+  <!-- ======================= -->
+  <!-- End the category for test goal -->
+  <!-- ======================= -->
+
   <!-- ======================= -->
   <!-- Setup the Root category -->
   <!-- ======================= -->
 
   <root>
-    <priority value="WARN"/>
-    <!-- <appender-ref ref="PENTAHOCONSOLE"/> -->
-    <appender-ref ref="PENTAHOFILE"/>
+    <priority value="DEBUG"/>
```

Comment:
This sets the category of everything defaulting to DEBUG. The root priority should remain WARN, and then you can override it for your specific classes you are testing.